SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -5,104 +5,230 @@
+
+            if (dockerCredentials.HasValue)
+            {
+                securityServiceContainer.WithCredential(dockerCredentials.Value.URL, dockerCredentials.Value.UserName, dockerCredentials.Value.Password);
+            }
+
+            // Now build and return the container                
+            IContainerService builtContainer = securityServiceContainer.Build().Start().WaitForPort($"5551/tcp", 30000);
+            Thread.Sleep(20000); // This hack is in till health checks implemented :|
+
+            AddEntryToHostsFile("127.0.0.1", containerName);
+
+            logger.LogInformation("Security Service Container Started");
+
+            return builtContainer;
+        }
+
+        private static IContainerService SetupSecurityServiceTestUIContainer(String containerName, 
+                                                                String securityServiceContainerName, 
+                                                                Int32 securityServiceContainerPort,
+                                                                INetworkService networkService,
+                                                                (String clientId, String clientSecret) clientDetails)
+        {
+            // Management API Container
+            IContainerService securityServiceTestUIContainer = new Builder().UseContainer().WithName(containerName)
+                                                                            .WithEnvironment($"Authority=http://{securityServiceContainerName}:{securityServiceContainerPort}",
+                                                                                             $"ClientId={clientDetails.clientId}",
+                                                                                             $"ClientSecret={clientDetails.clientSecret}")//,
+                                                                                             //$"MetadataAddress=http://{securityServiceContainerName}:{securityServiceContainerPort}/.well-known/openid-configuration")
+                                                                            .UseImage("securityservicetestwebclient").ExposePort(5004)
+                                                                            .UseNetwork(new List<INetworkService>
+                                                                                        {
+                                                                                            networkService
+                                                                                        }.ToArray())
+                                                                            .Build().Start();//.WaitForPort("5004/tcp", 30000);
+
+            return securityServiceTestUIContainer;
+
+            //Console.Out.WriteLine("Started Security Service");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)